src/tools/github2gitlab.rst

```diff
@@ -0,0 +1,73 @@
+=====================================
+Transition your Github Repo to Gitlab
+=====================================
+
+Create Gitlab Account
+=====================
+
+..
+  gitlab.com > create account
```

Comment:
We have instructions on how to create a GitHub account already for new devs - we should update that as well. If I recall, there's a requirement to either use your institutional email when creating your github/lab account, or at least to add that address to your profile.